app/sync.js

```diff
@@ -371,7 +371,13 @@ module.exports.init = function (appState) {
     reset()
   })
   // GET_INIT_DATA is the first message sent by the sync-client when it starts
-  ipcMain.on(syncMessages.GET_INIT_DATA, (e) => {
+  ipcMain.on(syncMessages.GET_INIT_DATA, (e, syncVersion) => {
+    if (syncVersion) {
+      appActions.addVersionInformation(Immutable.fromJS([{
```

Comment:
i think it should be `addVersionInformation` (like addBookmark, addSite, etc.) because the version information has not been added to the appState yet at this point